### PR TITLE
[Python Example] Fix compression example

### PR DESCRIPTION
**Pull request by XuanWang-Amos**
Fix: https://github.com/grpc/grpc/issues/34690

Also added generated proto files to example folder so we can run example directly.

### CODE REVIEW

File: examples/python/compression/client.py [hunk 1]
```diff
@@ -21,9 +21,8 @@
 import logging
 
 import grpc
-
-from examples.protos import helloworld_pb2
-from examples.protos import helloworld_pb2_grpc
+import helloworld_pb2
```
Comment (gnossen) [inline examples/python/compression/client.py:24]: I'm wondering how this works under Bazel. PYTHONPATH should be pointing to the repo root in that case. How are we picking up `helloworld_pb2` there?

I'm all for enabling running the example under both Bazel and non-Bazel, but we have to enable that deterministically.

File: examples/python/compression/client.py [hunk 2]
```diff
@@ -40,12 +39,13 @@ def run_client(channel_compression, call_compression, target):
         target, compression=channel_compression
     ) as channel:
         stub = helloworld_pb2_grpc.GreeterStub(channel)
-        response = stub.SayHello(
-            helloworld_pb2.HelloRequest(name="you"),
-            compression=call_compression,
-            wait_for_ready=True,
-        )
-        print("Response: {}".format(response))
+        for _ in range(10):
```
Comment (gnossen) [inline examples/python/compression/client.py:42]: Please hoist the `10` to a constant.

File: examples/python/compression/client.py (hunk 1 above)
Comment (XuanWang-Amos) [reply examples/python/compression/client.py:25]: Looks like in the Bazel test we're using `subprocess` to start client and server, so technically those scripts was started  at parent directory?
https://github.com/grpc/grpc/blob/352c33986e5069abbdc8496915d8f913230c15df/examples/python/compression/test/compression_example_test.py#L22-L24

I think that's why we have `examples/python/compression` added to PYTHONPATH.

Comment (gnossen) [reply examples/python/compression/client.py:24]: Resolved offline. This actually seems to be behavior provided by Bazel to emulate a script being run from its own directory. Surprising, but harmless.

**Review by gnossen — APPROVED** [review]

**Pull request merged**